### PR TITLE
rm: linux: Add howto for linux-lmp-fslc-imx-rt

### PR DESCRIPTION
**Pull request by angolini**
I decided to duplicate the same section, however I would like to hear in case of any better idea.

## Readiness

* [X] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [X] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [X] View HTML in a browser to check rendering.
* [X] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [X] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [X] End message with sign off/DCO line (`-s, --signoff`).
  * [X] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [X] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 7425ba4ce4edafa1383dae5eecdb56f46605fffa are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1807/docs/artifacts/html/index.html

**Review by kprosise — APPROVED** [review]
The addition looks fine. We could potentially move "Now just build..." to before the subsections, such as around line 45, with something like " Set the appropriate build variables as detailed below, then you can build the LmP image as usual".  But we could also tackle that later.

**Comment by angolini** [comment]
@kprosise I changed everything :smile: 
Please take a look at the new commit. If you like it, I squash

**Comment by doanac** [comment]
Docs for 0c6831a5bdd15c2e0ce0c1a9dac61b65a5ab2023 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1823/docs/artifacts/html/index.html

File: source/reference-manual/linux/linux-kernel.rst [hunk 1]
```diff
@@ -37,22 +37,36 @@ Linux microPlatform with Real-Time Linux Kernel
 -----------------------------------------------
 
 The recipe ``meta-lmp/meta-lmp-base/recipes-kernel/linux/linux-lmp-rt_git.bb``
-can be used for real-time linux, which is based on the ``linux-lmp`` recipe,
-but extended to include the PREEMPT_RT patch-set (updated along with stable
-kernel updates).
+or ``meta-lmp/meta-lmp-base/recipes-kernel/linux/linux-lmp-fslc-imx-rt_git.bb``
+can be used for real-time linux,
```
Comment (kprosise) [inline source/reference-manual/linux/linux-kernel.rst:41]:
```suggestion
can be used for real-time Linux.
```

File: source/reference-manual/linux/linux-kernel.rst [hunk 2]
```diff
@@ -37,22 +37,36 @@ Linux microPlatform with Real-Time Linux Kernel
...
+which is based on the ``linux-lmp`` recipe,
+but extended to include the PREEMPT_RT patch-set
```
Comment (kprosise) [inline source/reference-manual/linux/linux-kernel.rst:43]:
```suggestion
but extended to include the ``PREEMPT_RT`` patch-set
```

File: source/reference-manual/linux/linux-kernel.rst [hunk 3]
```diff
@@ -37,22 +37,36 @@ Linux microPlatform with Real-Time Linux Kernel
...
+in ``meta-subscriber-overrides/conf/machine/include/lmp-factory-custom.inc``::
 
     $ cat meta-subscriber-overrides/conf/machine/include/lmp-factory-custom.inc
     PREFERRED_PROVIDER_virtual/kernel:intel-corei7-64 = "linux-lmp-rt"
 
-Now just build the Linux microPlatform image as usual.
+Building Linux microPlatform with linux-lmp-fslc-imx-rt
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Set the ``PREFERRED_PROVIDER_virtual/kernel``
```
Comment (kprosise) [inline source/reference-manual/linux/linux-kernel.rst:64]:
```suggestion
Set ``PREFERRED_PROVIDER_virtual/kernel``
```

**Review by kprosise — APPROVED** [review]
Some small suggestions. After they are resolved, squash and merge at your convenience. :)

**Comment by doanac** [comment]
Docs for 72dffc9d54f33d478cd29a8e7aa37bf69114fe45 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1832/docs/artifacts/html/index.html

**Pull request merged**